pantograph/utils.py

```diff
@@ -0,0 +1,116 @@
+import os
+import os.path as osp
+import asyncio
+import warnings
+from pathlib import Path
+import collections as C
+import functools as F
+
+import pexpect
+import regex as re
+
+
+def to_sync(func):
+    @F.wraps(func)
+    def wrapper(*args, **kwargs):
+        return asyncio.get_event_loop().run_until_complete(func(*args, **kwargs))
+    return wrapper
+
+async def check_output(*args, **kwargs):
+    p = await asyncio.create_subprocess_exec(
+        *args, 
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+        **kwargs,
+    )
+    stdout_data, stderr_data = await p.communicate()
+    if p.returncode == 0:
+        return stdout_data
+
+class Spwan(pexpect.spawn):
```

Comment:
is it intentional that you spelt this with Spwan to avoid a name collision?

Reply:
I just casually came up with that name. If you think it's necessary to change, just go ahead :)